torch/distributed/rpc/api.py

```diff
@@ -259,35 +264,57 @@ def remote(to, func, args=None, kwargs=None):
     kwargs = kwargs if kwargs else {}
 
     info = _to_worker_info(to)
+    current_worker = get_worker_info()
+    # If profiling is enabled, kick off the timer and retrieve back a
+    # RecordFunction instance.
+    rf = None
+    if torch.autograd._profiler_enabled():
+        rf = _start_record_function(
```

Comment:
We should start the recording as the first line in this function (before `torch.jit._find_builtin`)

Reply:
For the name parameter passed into record_function though, we need the `qualified_name` in the case a builtin is being invoked. So we'd need to call `torch.jit_find_builtin` since the return value of that is needed to call `_start_record_function`.